test-integration/src/test/java/nl/nn/adapterframework/filesystem/AmazonS3SenderTest.java

```diff
@@ -78,6 +81,7 @@ public AmazonS3Sender createFileSystemSender(){
 		s3.setAccessKey(accessKey);
 		s3.setSecretKey(secretKey);
 		s3.setBucketName(bucketName);
+		s3.setProxyConfig(proxyHost, proxyPort);
 		return s3;
 	}
 
```

Comment:
Could you show in a test that it works both with and without a proxy configured?